Macros/Compile_ACS.sas

```diff
@@ -49,6 +50,27 @@
     %let geo_suffix = tr00;
     %let geo_var = Geo2000;
   %end;
+  %else %if &geo = CITY %then %do;
+    %** city level - use only for DC **;
+    %let _acs_sf_raw_path = &_acs_sf_raw_base_path\&_state_name._All_Geographies_Not_Tracts_Block_Groups;
+    %let sum_level = 040;
+    %let geo_suffix = city;
+    %let geo_var = city;
+  %end;
+  %else %if &geo = COUNTY %then %do;
```

Comment:
@ptatian @rpitingolo - it seems worth doing. but the really issue is that there are apparently a lot of boundary changes and not that many FIPS code changes. And unlike the ward2012 designation -which reflects a boundary change the county2010 (or whatever we decide the right year is wouldn't?) See for example all of the changes in Virginia: https://www.census.gov/geo/reference/boundary-changes.html